websites/A/Artfight/dist/metadata.json

```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Artfight",
 	"description": {
-		"en": "The annual art trading game, starting on July 1st!"
+		"en": "The annual art trading game, starting on July 1st!",
+		"fr": "L'annuel jeu d'échange d'art, commence le premier juillet !"
 	},
 	"url": "artfight.net",
-	"version": "1.0.0",
+	"version": "1.1.0",
```

Comment:
```suggestion
	"version": "1.0.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8